frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -1122,8 +1122,8 @@ end
 function ReaderHighlight:onHoldRelease()
     local long_final_hold = false
     if self.hold_last_tv then
-        local hold_duration = TimeVal.now() - self.hold_last_tv
-        hold_duration = hold_duration.sec + hold_duration.usec/1000000
+        local hold_duration = TimeVal:monotonic() - self.hold_last_tv
```

Comment:
Does this needs to be named `:monotonic()` ? When reading higher level code, `:now()` was a LOT MORE explicite and read like natural language.
"monotonic" might be some technical term/API that you use internally to get a TimeVal, but I think it should be kept internal, and higher level code to keep being readable :)

Reply:
The other option is making a gettime w/ a clockid parameter, which is possibly worse ;p.

Reply:
Was just thinking about another option: give it a better less technical name :)
`:rightnow()` :) 
or why not just `:now()` ? Do we still need the current `:now()` elsewhere ?

Reply:
Yeah, I'm going to check that, and possibly rename now to `realtime` and alias `now` to `monotonic` if that's viable ;).